cli/cache/incremental.rs

```diff
@@ -42,6 +43,19 @@ impl IncrementalCache {
     IncrementalCache(IncrementalCacheInner::new(db, state, initial_file_paths))
   }
 
+  #[allow(unused)]
+  pub fn new_with_state_cb<TState: std::hash::Hash>(
```

Comment:
Maybe we should just change `new` to need to accept a `CacheDBHash`? Then we don't need to code generate multiple paths.

Reply:
Let me try that

Reply:
PTAL again

Reply:
Did you mean to push?

Reply:
Yes, I did.